backend/src/test/java/com/woowacourse/friendogly/pet/domain/SizeTypeTest.java

```diff
@@ -0,0 +1,31 @@
+package com.woowacourse.friendogly.pet.domain;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.woowacourse.friendogly.exception.FriendoglyException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class SizeTypeTest {
+
+    @DisplayName("String 타입으로 SizeType을 입력하면 Enum으로 변환한다.")
+    @ValueSource(strings = {"SMALL", "MEDIUM", "LARGE"})
+    @ParameterizedTest
+    void toSizeTypre(String input) {
+        assertThatCode(() -> SizeType.toSizeType(input))
+                .doesNotThrowAnyException();
+    }
+
+    @DisplayName("존재하지 않는 SizeType을 입력하면 예외가 발생한다.")
+    @Test
+    void toSizeType_Fail_InvalidSizeType() {
+        String input = "INVALID_SIZE_TYPE";
+
+        assertThatThrownBy(() -> SizeType.toSizeType(input))
+                .isExactlyInstanceOf(FriendoglyException.class)
+                .hasMessage("존재하지 않는 SizeType 입니다.");
+    }
+}
```

Comment:
```suggestion
    }
}

```

IntelliJ의 저장 시 자동으로 EOF 처리해주는 옵션을 사용해 봐도 좋을 것 같아요.

Reply:
https://jojoldu.tistory.com/673